core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -123,8 +123,9 @@
   @Sensitive
   INSTANCE_SECRET("instance.secret", "DEFAULT", PropertyType.STRING,
       "A secret unique to a given instance that all servers must know in order to communicate with one another."
-          + " Change it before initialization. To change it later use ./bin/accumulo accumulo.server.util.ChangeSecret [oldpasswd] [newpasswd], "
-          + " and then update conf/accumulo-site.xml everywhere."),
+          + "Change it before initialization. To change it later use the ChangeSecret tool and then update conf/accumulo-site.xml everywhere."
```

Comment:
maybe `s/Change it before initialization/It should be changed prior to initializing Accumulo/` ?

[nits] `s/later use/later, use/`, `s/and then/and/`